python/ray/_private/memory_monitor.py

```diff
@@ -103,6 +103,8 @@ class MemoryMonitor:
 
     The environment variable `RAY_MEMORY_MONITOR_ERROR_THRESHOLD` can be used
     to overwrite the default error_threshold setting.
+
+    Used by test only. For production code use memory_monitor.cc
```

Comment:
Can we remove the `raise_if_low_memory` method?

Reply:
It is used by the mem_check.py script, which is used by agent_stress_test in our nightly

@rkooo567

Reply:
Actually, I'll leave this comment open to see if it can be removed from the nightly test 

@rkooo567  can you advise?

Reply:
Ah I see, didn't spot that.